test/common/router/scoped_rds_test.cc

```diff
@@ -70,7 +110,11 @@ class ScopedRdsTest : public ScopedRoutesTestBase {
 name: foo_scoped_routes
 scope_key_builder:
   fragments:
-    - header_value_extractor: { name: X-Google-VIP }
+    - header_value_extractor:
+        name: X-Google-VIP
```

Comment:
Let's rename this header to something more generic; e.g., "X-VIP".

Reply:
done